mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature

```diff
@@ -46,6 +46,94 @@ Feature: Test quickMARC holdings records
     Then match tag.content != null
     Then match tag.content contains "$b olin"
 
+  Scenario: Edit quick-marc record tags
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+
+    * def newTagContent = 'Updated Content' + random
+    * set record.fields[?(@.tag=='867')][0].content = newTagContent
+    * set record.relatedRecordVersion = 2
+
+    Given path 'records-editor/records', record.parsedRecordId
+    And headers headersUser
+    And request record
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And retry until response.updateInfo.recordState == 'ACTUAL'
+    And match response.fields[?(@.tag=='867')][0].content == newTagContent
+
+  Scenario: Edit quick-marc record remove not required tag
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+
+    * remove record.fields[?(@.tag=='867')]
+    * set record.relatedRecordVersion = 2
+
+    Given path 'records-editor/records', record.parsedRecordId
+    And headers headersUser
+    And request quickMarcJson
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And retry until response.updateInfo.recordState == 'ACTUAL'
+    And match response.fields[?(@.tag=='867')].length == 0
+
+  Scenario: Edit quick-marc record add new tag, should be updated in SRS
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+
+    * def fields = record.fields
+    * def newField = { "tag": "866", "indicators": [ "\\", "\\" ], "content": "$a Test note", "isProtected":false }
+    * fields.push(newField)
+
+    * set record.fields = fields
+    * set record.relatedRecordVersion = 2
+
+    Given path 'records-editor/records', record.parsedRecordId
+    And headers headersUser
+    And request quickMarcJson
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testHoldingsId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And retry until response.updateInfo.recordState == 'ACTUAL'
```

Comment:
I don't think this retry is actual. We have an updated flow for PUT endpoint and if it return 202 status then record is updated